### PR TITLE
Update roadmap for 2020 Q2

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation
> /kind test
> /kind failing-test
> /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/getting-started/roadmap_zh.md [hunk 1]
```diff
@@ -4,27 +4,34 @@
 
 [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) 包含了最新的开发进度
 
-KubeEdge 1.2 版本是我们目前的稳定版本， 在2019年Q4将添加如下的新特性。
+KubeEdge 1.3 版本是我们目前的稳定版本， 在2019年Q4将添加如下的新特性。
```
Comment (XJangel) [inline docs/getting-started/roadmap_zh.md:7]: I think maybe the 2019 Q4 should be 2020 Q2?

Comment (fisherxu) [reply docs/getting-started/roadmap_zh.md]: Done.

File: docs/getting-started/roadmap.md [hunk 2]
```diff
@@ -4,27 +4,35 @@ This document defines a high level roadmap for KubeEdge development.
...
-- Support for monitoring at the edge.
-- Support HA for cloudcore
-- Support exec&logs API for edge application
-- Add protobuf support for data exchange format between cloud and edge
-- Finish scalability test and publish report
-- Support managing clusters at edge from cloud (aka. EdgeSite)
-- Enhance performance and reliability of KubeEdge infrastructure.
-- Support ingress at edge.
-- Improve contributor experience by defining project governance policies, release process, membership rules etc.
-- Improve the performance and e2e tests with more metrics and scenarios.
```
Comment (chendave) [inline docs/getting-started/roadmap.md:19]: I'd like these info is kept here instead of being removed.

File: docs/getting-started/roadmap.md [hunk 3]
```diff
@@ -4,27 +4,35 @@ This document defines a high level roadmap for KubeEdge development.
...
+- Improve contributor experience by defining project governance policies, release process, membership rules etc.
+- Improve the performance and e2e tests with more metrics and scenarios.
+- Add protobuf support for data exchange format between cloud and edge
+- Finish scalability test and publish report
+- Support managing clusters at edge from cloud (aka. EdgeSite)
+- Enhance performance and reliability of KubeEdge infrastructure.
 - Support edge-cloud communication using edgemesh.
 - Istio-based service mesh across Edge and Cloud where micro-services can communicate freely in the mesh.
 - Enable function as a service at the Edge.
-- Support more types of device protocols such as OPC-UA, Zigbee.
```
Comment (chendave) [inline docs/getting-started/roadmap.md:27]: why this is removed? included here "Device CRD improvement, support device protocol extension." ?

**Comment by chendave** [comment]
/lgtm

As a alternative, we can keep all the legacy roadmap in this page, it's might be helpful for us to check the history.

**Comment by fisherxu** [comment]
> As a alternative, we can keep all the legacy roadmap in this page, it's might be helpful for us to check the history.

Good idea, we can further discuss how to present the roadmap. 
/cc @kevin-wangzefeng @kadisi @subpathdev

File: docs/getting-started/roadmap.md (hunk 3 above)
Comment (kevin-wangzefeng) [reply docs/getting-started/roadmap.md:27]: Yes

File: docs/getting-started/roadmap.md (hunk 2 above)
Comment (fisherxu) [reply docs/getting-started/roadmap.md:19]: Haven't removed them, just move to future list instead of Q2.

File: docs/getting-started/roadmap.md [hunk 4]
```diff
@@ -4,27 +4,35 @@ This document defines a high level roadmap for KubeEdge development.
...
+- Support list-watch from edgecore for applications on the edge.
+- Collect data information sent from the edge side from CloudHub
 - Improve KubeEdge installation experience
 - Add more docs and move docs out of main repo
 
+
 ## Future
+
+- Improve contributor experience by defining project governance policies, release process, membership rules etc.
+- Improve the performance and e2e tests with more metrics and scenarios.
```
Comment (chendave) [inline docs/getting-started/roadmap.md:28]: As discussed with fei xu,  arm e2e might be included in this bullet,  let me do some investigation first.

Comment (fisherxu) [reply docs/getting-started/roadmap.md:28]: OK, I think add `Add integration and e2e tests on ARM in CI` in future list?

Comment (chendave) [reply docs/getting-started/roadmap.md:28]: +1 will talk with you on this later.

**Pull request merged**